app/src/main/kotlin/dev/patrickgold/florisboard/lib/compose/ScrollableModifiers.kt

```diff
@@ -179,3 +251,20 @@ fun Modifier.florisScrollbar(
         }
     }
 }
+
+/**
+ * Item's offset on main axis as a percentage of size
+ */
+fun percentOffset (
+    item: LazyListItemInfo,
+): Float {
+    return item.offset.toFloat() / item.size
+}
+fun percentOffset (
```

Comment:
Please add an empty line here
```suggestion

fun percentOffset (
```